text/0018-bee-bundle/0018-bee-bundle.md

```diff
@@ -0,0 +1,308 @@
++ Feature name: `bee-bundle`
++ Start date: 2019-10-02
++ RFC PR: [iotaledger/bee-rfcs#18](https://github.com/iotaledger/bee-rfcs/pull/18)
++ Bee issue: [iotaledger/bee#63](https://github.com/iotaledger/bee/issues/63)
+
+# Summary
+
+This RFC is based on [`transaction-module`]().
+<!-- TODO FIx link to RFC#3 -->
+
+The smallest communication unit in the IOTA protocol is the transaction. Everything, including payment settlements
+and/or plain data, is propagated through the IOTA network in transactions.
+
+A transaction is `2673` trytes and the part available to the user is `2187` trytes. This part holds a signature in case
```

Comment:
* maybe we should also mention the byte equivalent? ~1605 bytes (in the most compact encoding)
* "the part available to the user" -> should we just use the common term "payload" to describe that?

Reply:
Janis wants trits, you want bytes, Yu-Wei wants trytes... :trollface: 
Thanks, I like payload.

Reply:
regarding the byte equivalent: I had my IOTA-newcomer glasses on :smile:

Reply:
If we go for something exotic, then it should be `trits`. `trytes` being defined in terms of `trits` is an extra layer on top of `trits`, so a user would need to keep both `trits` and `trytes` in mind.

I very much like bytes as well, but since then we also need to talk about compact encodings we should probably not do that.